Resources/Textures/Objects/Weapons/Grenades/tearsmokegrenade.rsi/meta.json

```diff
@@ -0,0 +1,21 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Created by Cthulhuwu (Discord)",
```

Comment:
If you are using someone's Discord as a copyright, you should contain their discrim

Reply:
discri-what?

Reply:
Four digits after #

Reply:
> Four digits after #

After the discord update, the usernames are no longer numbered...

Reply:
1. It's not out yet
2. Discord will still support the discrims for some time afaik